### PR TITLE
feat: Create grpc base for event service

### PR DESCRIPTION
**Pull request by ZixMai**
Created grpc server
Created event to grpc event converter
Created grpc model
Created grpc service implementation
Created docstrings to each def and class

### CODE REVIEW

File: event_service/src/event_service_impl.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+)
+from src.models.event import Event
+from datetime import datetime
+
+
+class EventServiceImpl(GrpcServicer):
+    """
+    Implementation of the Event Service.
+
+    ...
```
Comment (Nhsdkk) [inline event_service/src/event_service_impl.py]: Delete unnecessary dots here

File: event_service/src/event_service_impl.py [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    """
+
+    def __init__(self):
+        self.events = [
+            Event(
+                id="1",
+                title="Sample Event",
+                description="This is a sample event.",
+                color="blue",
+                start=datetime(2024, 3, 19, 10, 0),  # Example start time
```
Comment (Nhsdkk) [inline event_service/src/event_service_impl.py]: Delete unnecessary comments

File: event_service/src/event_service_impl.py [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+        Parameters
+        ----------
+        request : DeleteEventRequest
+            Request data containing event ID.
+        context : grpc.ServicerContext
+            Request context.
+
+        Returns
+        -------
+        StandardResponse
```
Comment (Nhsdkk) [inline event_service/src/event_service_impl.py]: Rename StandardResponse to BaseResponse for convenience

File: event_service/src/models/event.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+from datetime import datetime
+from proto.event_service_pb2 import Event as GrpcEvent
+
+
+@dataclass
+class Event:
+    """
+    Class with data about the event.
+
+    ...
```
Comment (Nhsdkk) [inline event_service/src/models/event.py]: Delete unnecessary dots here

File: event_service/proto/event_service.proto [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  google.protobuf.Timestamp end = 6;
+  optional google.protobuf.Timestamp repeating_delay = 7;
+  string author_id = 8;
+}
+
+message ListOfEvents {
+  repeated Event events = 1;
+}
+
+message StandardResponse {
```
Comment (Nhsdkk) [inline event_service/proto/event_service.proto]: Rename StandardResponse to BaseResponse for convenience

**Review by Nhsdkk — CHANGES_REQUESTED** [review]
Check my comments

**Review by Nhsdkk — APPROVED** [review]
LGTM

File: event_service/pyproject.toml [hunk 6]
```diff
@@ -3,13 +3,13 @@ name ="event_service"
 version = "0.1.0"
 description = ""
 authors = ["Nhsdkk <38581029+Nhsdkk@users.noreply.github.com>"]
```
Comment (MaxPlays35) [inline event_service/pyproject.toml]: ~~Add other authors please~~

File: event_service/src/event_service_impl.py [hunk 7]
```diff
@@ -0,0 +1,140 @@
+"""Event Service Controller"""
+from proto.event_service_pb2_grpc import EventServiceServicer as GrpcServicer
+from proto.event_service_pb2 import (
```
Comment (MaxPlays35) [inline event_service/src/event_service_impl.py]: Import using asterisk or 
```python
import proto.event_service_pb2 as proto
```

**Review by MaxPlays35 — CHANGES_REQUESTED** [review]

**Review by MaxPlays35 — APPROVED** [review]

**Review by MaxPlays35 — APPROVED** [review]

**Review by Nhsdkk — APPROVED** [review]
LGTM

**Pull request merged**